Suite2p/suite2p_parameters.py

```diff
@@ -23,12 +23,18 @@ def __init__(self, danse_params: dict):
         self.time_length: int      = self.get_time_length()
         
         self.ops = suite2p.default_ops()
-        self.ops['batch_size']        = 50 # Decrease the batch_size in case low RAM on computer
-        self.ops['threshold_scaling'] = self.params['Threshold'] # Threshold for ROIs detection
-        self.ops['tau']               = self.params['DecayTime'] # Timescale of gcamp to use for deconvolution
+        self.ops['threshold_scaling'] = self.params['ROIsThreshold'] # Threshold for ROIs detection
+        self.ops['tau']               = self.params['BiosensorDecayTime'] # Timescale of gcamp to use for deconvolution
+        self.ops['diameter']          = self.params['Diameter'] # (cellpose)
+
         self.ops['nplanes']           = len(self.paths[defs.Parameters.Path.H5PATH])
         self.ops['data_path']         = [self.paths[defs.Parameters.Path.TMP_DIR]]
 
+        self.ops['batch_size']        = 50 # Decrease the batch_size in case low RAM on computer
```

Comment:
I'm not sure about that.. If the computer is good, the processing would be just long for nothing

Reply:
Yepp, for now I'd keep the default 500. We can try increase/decrease this value depending on the RAM later

Reply:
500 is too much for voluscan :/ 

100 is good on my machine. I can check for 200 how it go

Reply:
oki let's update to 100